linetools/isgm/utils.py

```diff
@@ -411,6 +411,12 @@ def table_from_complist(complist, summed_ion=False, NHI_obj=None, vrange=None,
     # Rename
     tab.rename_column('zcomp', 'z_comp')
 
+    # logN attribs, they should be stored only in the icomp.attrib dict()
+    for ak in  ['flag_N', 'logN', 'sig_logN']:
+        values = [icomp.attrib[ak] for icomp in complist]
+        import pdb; pdb.set_trace()
```

Comment:
Need to remove this

Reply:
I think you should add `flag_N`, `logN` etc to the list of 
attributes that occur a few lines down (e.g. including `b`, `sig_b`).
No need for a separate for loop here.

Reply:
You are right. Modified accordingly.